ksql-engine/src/main/java/io/confluent/ksql/engine/EngineExecutor.java

```diff
@@ -115,7 +118,10 @@ ExecuteResult execute(final ConfiguredStatement<?> statement) {
     }
   }
 
-  private void validateQuery(final QueryMetadata query, final ConfiguredStatement<?> statement) {
+  private void validateQuery(
+      final QueryMetadata query,
+      final ConfiguredStatement<?> statement
+  ) throws IOException, RestClientException {
```

Comment:
We shouldn't be throwing checked exceptions here.  Any `IOException` should be caught internally and a better error message built.  (Though I don't think these exceptions are being thrown in this instance)

```suggestion
  ) {
```